packages/tempus/src/Tempus_TimeEventBase.hpp

```diff
@@ -38,79 +38,189 @@ class TimeEventBase
 
   /// Constructor
   TimeEventBase()
-    : name_("TimeEventBase"),
-      defaultTime_ (-std::numeric_limits<Scalar>::max()*1.0e-16),
-      defaultTol_  ( std::numeric_limits<Scalar>::min()),
-      defaultIndex_( std::numeric_limits<int>::min())
+    : timeEventType_("Base"),
+      name_("TimeEventBase"),
+      defaultTime_ (std::numeric_limits<Scalar>::max()),
+      defaultTol_  (std::numeric_limits<Scalar>::min()),
```

Comment:
use `epsilon()` instead?

Reply:
So I think the question is, is defaultTol_ a relative tolerance or absolute tolerance?  I think when I started it was an absolute tolerance, but now it is not used at all?!  So to make it type independent, I think we should go with epsilon(), and use defaultTol_ to set the default relative tolerances.

Reply:
Done!